server/server.js

```diff
@@ -370,7 +371,107 @@ function startServer() {
   );
 }
 
+async function fetchCitybikeSeasons() {
+  const client = new CosmosClient(process.env.CITYBIKE_DB_CONN_STRING);
+  const database = client.database(process.env.CITYBIKE_DATABASE);
+  const container = database.container('schedules');
+  const query = {
+    query: 'SELECT * FROM c',
+  };
+
+  const { resources } = await container.items.query(query).fetchAll();
+  return resources;
+}
+
+const parseDate = (year, month, day) =>
+  // eslint-disable-next-line radix
+  new Date(parseInt(year), parseInt(month) - 1, parseInt(day));
+
+function buildCitybikeConfig(cb, networkName, configName) {
+  const inSeason = cb.inSeason.split('-');
+  const [startDay, startMonth, startYear] = inSeason[0].split('.');
+  const [endDay, endMonth, endYear] = inSeason[1].split('.');
+  const [preDay, preMonth, preYear] = cb.preSeason.split('.');
+  return {
+    config: configName,
+    networkName,
+    region: cb.region,
+    enabled: cb.enabled,
+    season: {
+      preSeasonStart: parseDate(preYear, preMonth, preDay),
+      start: parseDate(startYear, startMonth, startDay),
+      end: parseDate(endYear, endMonth, endDay),
+    },
+  };
+}
+
+function handleCitybikes(schedules, configName) {
+  const bikes = schedules.filter(
+    cb => cb.region.toLowerCase() === configName || cb.config === configName,
+  );
+  const cbConfigurations = [];
+  bikes.forEach(bike => {
+    // This check is due to naming scheme in HSL citybike networks (Smoove and Vantaa)
+    if (bike.config && bike.config === 'hsl') {
```

Comment:
can bike.config be falsy?

Reply:
Yes, currently the config field is extra information for convenience in HSL citybikes. Real question is should it be falsy.

Let's hit with a call tomorrow and discuss this in more detail. I would like some insight on this one.